sourcegraph/sourcegraph
# Wildcard: Tabs - add scrolling fade-out gradients

As pointed out in the [previous PR](https://github.com/sourcegraph/sourcegraph/pull/34765), it would be great to have not only scrolling but also a visual hint of the fact that it's scrollable.

This PR adds gradients as suggested


https://user-images.githubusercontent.com/2196347/169811100-82f55c72-3062-4dba-82a9-698ae7634b1c.mov



## Test plan
1. Pull the branch
2. Run wildcard storybook: `yarn storybook:wildcard`
3. Check the component in light and dark themes

## App preview:

- [Web](https://sg-web-og-tabs-scroll-gradient.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-pdwokssorr.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
